doc/user_guide/configuration/engine_configuration.rst

```diff
@@ -0,0 +1,157 @@
+    # Additional parameters are preceded by a `&`.
+    query = "?AUTOCOMMIT=y"
+
+    url_string = f"exa+websocket://{username}:{password}@{host}:{port}/{schema}{query}"
+
+    create_engine(url_string)
+
+.. _suggested_parameters:
+
+Suggested Parameters
+--------------------
+
+In the table below are selected parameters that are suggested for specific scenarios.
+The examples are given for :ref:`instance_url` where ``query`` is a keyword
+argument supplied to the ``URL.create()`` constructor.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Example
+     - Description
+   * - CONNECTIONLCALL
+     - .. code-block:: python
+
+         query={
+          "CONNECTIONLCALL": "en_US.UTF-8"
+         }
+     - To avoid errors due to different code pages used by the client process (Python)
+       and the Exasol driver, this is recommended to use, particular for Unix-based
+       systems.
+
+.. _dialect_specific_params:
+
+Dialect-Specific Parameters
+---------------------------
+In the table below are additional parameters that are specific to the SQLAlchemy-Exasol
+dialect. The examples are given for :ref:`instance_url` where ``query`` is a keyword
+argument supplied to the ``URL.create()`` constructor.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Example
+     - Description
+   * - AUTOCOMMIT
+     - .. code-block:: python
+
+         query={
+          "AUTOCOMMIT": "y"
+         }
+     - This indicates if the connection should automatically commits or not.
+       The parsed value is case-insensitive.
+
+        * To enable autocommit, specify ``"y"`` or ``"yes"``.
+        * To disable autocommit, specify ``"n"`` or ``"no"``.
+
+       The default is for autocommit to be enabled (``"y"``).
+   * - ENCRYPTION
+     - .. code-block:: python
+
+         query={
+          "ENCRYPTION": "y"
+         }
+     - This indicates if the connection should be encrypted or not.
+       The parsed value is case-insensitive.
+
+        * To enable TLS encryption, specify ``"y"`` or ``"yes"``.
+        * To disable TLS encryption (not recommended), specify ``"n"`` or ``"no"``.
+
+       The default is for TLS encryption to be enabled (``"y"``). For more information
+       about TLS encryption, please see :ref:`tls`.
+   * - FINGERPRINT
+     - .. code-block:: python
+
+         query={
+          "FINGERPRINT": "<fingerprint>"
+         }
+     - An alternate to SSL certificate verification is to verify the connection
+       via a fingerprint.
+
+        * By default, fingerprint verification is not active.
+        * To use fingerprint verification, provide your fingerprint value
+          (i.e. "0ACD07D4E9CEEB122773A71B9C3BD01CE49FC99901DE7C0E0030C942805BA64C").
+
+       For more information about fingerprint verification, please see
+       :ref:`fingerprint_verification`.
+   * - SSLCertificate
+     - .. code-block:: python
+
+         query={
+          "SSLCertificate": "SSL_VERIFY_NONE"
```

Comment:
I think this is a bit inconsistent, as the other boolean options are using "y" and "n", while this one uses
* default / not specified vs. 
* `SSL_VERIFY_NONE`

Would it be a breaking change, changing this?

Another question is: How can the user specify explicitly to verify the certificates?
Aside from using the default / not specified: Is there an explicit value as well? 
(This use case could be called "Re-enabling the certificate validation")